Documentation/general/tool-nuget-package-format.md

```diff
@@ -0,0 +1,28 @@
+Tool NuGet package format
+-------------------------------------------
+
+The most straightforward way to create a .NET tool package is to run `dotnet pack` with `PackAsTool` property set in the project file. However, if your build process is highly customized, `dotnet publish` may not create the right package for you. In this case, you can create a NuGet package for your tool using a *nuspec* file and explicitly placing assets into the NuGet package following these rules.
+
+- The NuGet package has only _/tools_ folder under the root and does **not** contain any other folders; do not include folders like  _/lib_, _/content_, etc.
+- Under _/tools_ folder, the subfolders must be structured with pattern _target framework short version/RID_. For example, tool assets targeting .NET core framework V2.1 that are portable across platforms should be in the folder _tools/netcoreapp2.1/any_.
+
+Let's call assets under every _tools/target framework short version/RID_ combination per TFM-RID assets :
```

Comment:
Kathleen:
- Under _/tools_ folder, the subfolders must be structured with pattern _target framework short version/RID_. For example, tool assets targeting .NET core framework V2.1 that are portable across platforms should be in the folder _tools/netcoreapp2.1/any_. 

I do not understand what you are trying to say in the last sentence:

Let's call assets under every _tools/target framework short version/RID_ combination per TFM-RID assets.

---------------
Will:
I try to make a definition 
per TFM-RID asset = assets under every tools/target framework short version/RID combination.

Reply:
Suggest quotes around "per TFM-RID assets"